en/SetupView/Motors_ardusub.md

```diff
@@ -1,13 +1,27 @@
 # Motors Setup (ArduSub)
 
+## Manual Test
 The ArduSub motor setup allows you to test individual motors.
 The sliders allow spinning each motor in forward or reverse mode, and the checkboxes under the sliders allow reversing the operation of individual thrusters.
 
 The image at the right shows the frame currently in use, along with the location and orientation of each thruster. 
 If the frame selection does not match your vehicle, first select the correct frame in the [Frame](../SetupView/airframe_ardupilot.md#ardusub) tab.
 
-To set up and test the motors, read and follow the instructions on the page.
+To manually set up and test the motors, read and follow the instructions on the page.
 
 > **Warning** Make sure the motors and propellers are clear from obstructions before sliding the switch to arm the vehicle and enable the test!
 
 ![Ardusub Motors Test](../../assets/setup/motors-sub.jpg)
+
+## Automatic Direction Detection
```

Comment:
What do you want people to do? If the automatic testing is the recommended and easiest approach then suggest you put docs for it first.

I also quite like direction at the top - e.g below main heading tell people their options and why each might be selected.

Reply:
Thank you for the tips.
I added a section below the main heading explaining how it works.